modules/Model/PostModel.php

```diff
@@ -17,19 +17,31 @@ class PostModel extends Module
 
     public function init(): void
     {
-        add_filter('timber/post/pre_meta', [$this, 'transformDateMeta']);
+        add_filter('timber/post/pre_meta', [$this, 'transformDateMeta'], 99, 3);
```

Comment:
I think there's a simpler solution here. Let's try using the same ACF filters as Timber, but at a lower priority: 
https://github.com/timber/timber/blob/2.x/src/Integration/AcfIntegration.php#L281-L282
Then the `transformDateMeta` function should be able to stay the same.